docs/user-guide/install.md

```diff
@@ -6,18 +6,92 @@ vSphere Docker Volume Service is simple to install. It has zero configuration an
 
-<script type="text/javascript" src="https://asciinema.org/a/80405.js" id="asciicast-80405" async></script>
+```
+# esxcli software vib install -v /tmp/vmware-esx-vmdkops-0.12.ccfc38f.vib
+Installation Result
+   Message: Operation finished successfully.
+   Reboot Required: false
+   VIBs Installed: VMWare_bootbank_esx-vmdkops-service_0.12.ccfc38f-0.0.1
+   VIBs Removed:
+   VIBs Skipped:
+```
 
 # Install on VM
 
-We currently package the service as a RPM and Deb package. This is to be able to start the service before Docker engine starts. We will also support Docker plugin framework once it is ready.
+**vDVS managed plugin**
+* **Prerequisite**: Docker 1.13/17.03 and above
 
+vDVS 0.13 comes with managed plugin support using Docker plugin framework.
+
+**vDVS managed plugin life cycle**
+
+* **Installation**
+```
+~# docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest
+0.12: Pulling from vmware/docker-volume-vsphere
+27de4a000433: Download complete
+Digest: sha256:696017e620fe8d29c883f36b5676135eaa2c2b3c3810f82931efb8027369a186
+Status: Downloaded newer image for vmware/docker-volume-vsphere:0.12
+Installed plugin vmware/docker-volume-vsphere:0.12
+```
+
+Let's verify the installed plugin using `docker plugin ls` command. In following example, you can see that vDVS plugin is not enabled yet.
+
+**Note**: After installation, the plugin needs to be enabled explicitely.
+
+```
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+de30e65cda7b        vsphere:latest      VMWare vSphere Docker Volume plugin   false
+```
+
+* **Enabling/Disabling plugin**
+
+Using `docker plugin enable/disable <plugin_name>`
+
+```
+~# docker plugin enable vsphere
+ vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+de30e65cda7b        vsphere:latest      VMWare vSphere Docker Volume plugin   true
+```
+
+Now plugin is enabled and ready to use (The `docker volume` command behaves as before and driver name should be what we passed as `--alias` )
+
+```
+~# docker plugin disable vsphere
+vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+de30e65cda7b        vsphere:latest      VMWare vSphere Docker Volume plugin   false
+```
+
+* **Removing plugin**
+
+Using `docker plugin rm <plugin_name>`
+
+```
+~# docker plugin rm vsphere
+vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION         ENABLED
+```
+
+**vDVS DEB/RPM based installation**
+
+**Note** DEB/RPM packages will be deprecated going forward and will not be available. It is recommeded upgrading Docker 1.13/17.03 and above and install vDVS managed plugin.
```

Comment:
You can drop "it is recommeded upgrading Docker 1.13/17.03 and above and install vDVS managed plugin."